app/views/shared/components/_fixed_side_bar.html.erb

```diff
@@ -24,7 +24,7 @@
           <% end %>
         </li>
         <li class="side-bar-item">
-          <%= link_to "#", class: "side-bar-item-inner" do %>
+          <%= link_to manager_path, class: "side-bar-item-inner" do %>
```

Comment:
can you skip this change for now ? I will update this separately. We need to make this change for mobile sidebar too.

Reply:
reverted